document_merge_service/api/unoconv.py

```diff
@@ -0,0 +1,22 @@
+import io
+
+import requests
+from django.conf import settings
+from rest_framework import exceptions, status
+
+
+def convert(from_file, to_type):
+    """
+    Use docker-unoconv-webservice to convert document to given type.
+
+    Returns `io.BytesIO` of file or raises `exceptions.ParseError` when conversion failed.
+
+    See: https://github.com/zrrrzzt/tfk-api-unoconv
+    """
+    url = f"{settings.UNOCONV_URL}/unoconv/{to_type}"
+    response = requests.post(url, files={"file": from_file})
+
+    if response.status_code == status.HTTP_200_OK:
+        return io.BytesIO(response.content)
```

Comment:
I think it would be a good idea to just return the full response in the success case. This way, you can also return the response from the view, taking advantage of streaming (you don't read the buffer, then write it again)

Reply:
very valid point. Problem is though that we have a requests response here but view expects a django response.

I have adjusted that this conversion actually happens. Also I got rid of all other BytesIO buffers which were not really necessary.